server/tests/src/test/java/org/infinispan/server/security/authorization/AbstractAuthorization.java

```diff
@@ -1,5 +1,6 @@
 package org.infinispan.server.security.authorization;
 
+import static junit.framework.TestCase.assertNotNull;
```

Comment:
Any reason using this import instead of `org.junit.jupiter.api.Assertions.assertNotNull`?

Reply:
No that was a mistake, good spot!